clion/build.gradle.kts

```diff
@@ -9,11 +9,16 @@ intellij {
 dependencies {
     implementation(project(":common"))
     implementation("org.sonarsource.sonarlint.core:sonarlint-core:$sonarlintCoreVersion")
-    testImplementation(platform("org.junit:junit-bom:5.7.1"))
-    testImplementation("org.junit.jupiter:junit-jupiter")
-    testImplementation("org.mockito:mockito-core:3.10.0")
+    testImplementation(libs.junit.api)
+    testRuntimeOnly(libs.junit.engine)
+    // Needed for https://github.com/gradle/gradle/issues/22333
+    testRuntimeOnly(libs.junit.launcher)
+    testImplementation(libs.mockito.core)
 }
 
 tasks.test {
     useJUnitPlatform()
+    testLogging {
+        events("passed", "skipped", "failed")
```

Comment:
I didn't know about this property. Does it print a summary at the end of tests execution ? I looked in the CI logs and I didn't find any additional logs.

I would be in favor of activating such traces on all projects, I find the Gradle test task too silent about what was run

Reply:
I tried to add this when investigating JUnit runtime error, but it was not useful, so I suggest to remove it for now. I agree that Gradle test task is too silent, but this is another topic ;)